commit 0a02be907c00ff61559a6c1335e109e27c7d42aa
Author: Lucas Alvares Gomes <lucasagomes@gmail.com>
Date:   Wed Dec 11 15:17:58 2019 +0000

    Add "OVN" debs and rpms package files
    
    This commit is adding the package files for OVN.
    
    Change-Id: I92ab727d9954eb729c41b9a67ecb60b56883097b
    Signed-off-by: Lucas Alvares Gomes <lucasagomes@gmail.com>

diff --git a/files/debs/ovn b/files/debs/ovn
new file mode 100644
index 00000000..81eea5e6
--- /dev/null
+++ b/files/debs/ovn
@@ -0,0 +1,3 @@
+ovn-central
+ovn-controller-vtep
+ovn-host
diff --git a/files/rpms/ovn b/files/rpms/ovn
new file mode 100644
index 00000000..698e57b0
--- /dev/null
+++ b/files/rpms/ovn
@@ -0,0 +1,3 @@
+ovn-central
+ovn-host
+ovn-vtep
